src/vmm/src/cpu_config/x86_64/cpuid/intel/normalize.rs

```diff
@@ -233,11 +233,18 @@ impl super::IntelCpuid {
 
     /// Update brand string entry
     fn update_brand_string_entry(&mut self) -> Result<(), NormalizeCpuidError> {
+        /// We always use this brand string.
+        const DEFAULT_BRAND_STRING_BASE: &[u8; 28] = b"Intel(R) Xeon(R) Processor @";
         // Get host brand string.
         let host_brand_string: [u8; BRAND_STRING_LENGTH] = host_brand_string();
+        let mut zero_padded_default_brand_string: [u8; BRAND_STRING_LENGTH] =
+            [0; BRAND_STRING_LENGTH];
+        zero_padded_default_brand_string[..DEFAULT_BRAND_STRING_BASE.len()]
```

Comment:
nit: 
```suggestion
        zero_padded_default_brand_string[..DEFAULT_BRAND_STRING_BASE.len() - 2]
```
so that the `@` is not contained in the brand string if no frequency is found (this was the old behavior)

Reply:
Done.